zh/editor/publish/publish-openharmony.md

```diff
@@ -14,25 +14,26 @@
 
 ## 准备工作
 
-1. 下载 Cocos Creator 3.6.1-OH 传送门：[Cocos Creator 3.6.1-OH](https://download.cocos.org/CocosCreator/voh/CocosCreator-v3.6.1-oh-win-092319.zip)
-
-2. 下载 OpenHarmany系统，下载地址：
+1. 下载 Cocos Creator 3.8.0-alpha 传送门：[Cocos Creator 3.8.0-alpha](https://download.cocos.org/CocosCreator/voh/CocosCreator-v3.6.1-oh-win-092319.zip)
 
+2.  下载 OpenHarmany系统与SDK 传送门：
 ```
-http://download.ci.openharmony.cn/version/Master_Version/OpenHarmony_3.2.5.5/20220926_081431/version-Master_Version-OpenHarmony_3.2.5.5-20220926_081431-dayu200.tar.gz
+https://gitee.com/openharmony/docs/blob/master/zh-cn/release-notes/OpenHarmony-v3.2-release.md#%E4%BB%8E%E9%95%9C%E5%83%8F%E7%AB%99%E7%82%B9%E8%8E%B7%E5%8F%96
 ```
-
-3. 下载 OpenHarmony SDK系统，下载地址：
-
+![](./publish-openharmony/document_image_rId2.png)
+OpenHarmony发布的系统是**32位**的，**64位**下载传送门：
```

Comment:
So need 32bit or 64 bit?

Reply:
Rk3568 supports 32-bit and 64-bit system, users can choose to download 32-bit or 64-bit system.